client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-detail/account-detail.component.html

```diff
@@ -42,28 +42,32 @@ <h2>{{ user?.full_name }}</h2>
     <ng-template #editView let-form="form">
         <h2>{{ 'Organization specific information' | translate }}</h2>
         <div [formGroup]="form">
-            <!-- Administration roles (at organization level) -->
-            <div *osOmlPerms="OML.can_manage_users; allowCommitteeAdmin: true">
-                <mat-form-field class="form100 force-min-with">
-                    <mat-label>{{ 'Administration roles (at organization level)' | translate }}</mat-label>
-                    <mat-select formControlName="organization_management_level">
-                        @if (!orgaManagementLevelChangeDisabled) {
-                            <mat-option [value]="null">-</mat-option>
-                        }
-                        @for (level of organizationManagementLevels; track level) {
-                            <mat-option [disabled]="orgaManagementLevelChangeDisabled" [value]="level">
-                                {{ getOmlVerboseName(level) | translate }}
-                            </mat-option>
-                        }
-                    </mat-select>
+            <!-- Home Committee -->
+            <div class="home-committee-area">
+                <mat-form-field class="form50 force-min-width">
+                    <mat-label>{{ 'Home committee' | translate }}</mat-label>
+                    <os-repo-search-selector
+                        formControlName="home_committee_id"
+                        [disabled]="!canManageHomeCommittee"
+                        [includeNone]="true"
+                        [multiple]="false"
+                        [repo]="committeeController"
+                        [sortService]="committeeSortService"
+                        [subscriptionConfig]="committeeSubscriptionConfig"
+                        [transformPropagateFn]="getTransformPropagateFn()"
+                        [transformSetFn]="getTransformSetFnHomeCommittee()"
+                    ></os-repo-search-selector>
                 </mat-form-field>
+                <mat-checkbox class="form50 guest-margin-top force-min-width" formControlName="guest">
+                    {{ 'guest' | translate }}
+                </mat-checkbox>
             </div>
             <!-- Committee admin -->
             <div *osOmlPerms="OML.can_manage_organization">
-                <mat-form-field class="form100 force-min-with">
+                <mat-form-field class="form100 force-min-width">
                     <mat-label>{{ 'Committee admin' | translate }}</mat-label>
                     <os-repo-search-selector
-                        formControlName="committee_management_ids"
+                        [formControl]="form.controls.committee_management_ids"
```

Comment:
I don't see why this needs to be changed

Reply:
Change not needed, reverted.